libs/shared/top-right-bar/src/lib/shared-top-right-bar.tsx

```diff
@@ -1,30 +1,97 @@
-import React, { ReactNode } from 'react';
+import React, { useState, MouseEvent, ReactNode } from 'react';
 import Toolbar from '@material-ui/core/Toolbar';
 import IconButton from '@material-ui/core/IconButton';
 import AccountCircle from '@material-ui/icons/AccountCircle';
 import SettingsIcon from '@material-ui/icons/Settings';
 import { Link } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import Badge from '@material-ui/core/Badge';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+
 import './shared-top-right-bar.scss';
 
+export interface Items {
+  _id: string;
+  title: string;
+  path: string;
+  timeStamp: string;
+  isRead: boolean;
+}
+
+export interface SharedTopMenuButtonProps {
+  items?: Items[];
+  children: ReactNode;
+}
+
 export interface MenuIconLink {
   _id: string;
   label: string;
   path: string;
 }
 
 export interface SharedTopRightBarProps {
-  dataFromDb: {
+  dataFromDb?: {
     settings: MenuIconLink;
     profile: MenuIconLink;
+    notifications: Items[];
   };
-  children: ReactNode;
 }
 
+const SharedTopMenuButton = (props: SharedTopMenuButtonProps) => {
+  const [menuOpen, setMenuOpen] = useState<boolean>(false);
+  const [anchorEl, setAnchorEl] = useState<HTMLElement>();
+
+  const handleMenu = (event: MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+    setMenuOpen(true);
+  };
+
+  const handleClose = () => {
+    setMenuOpen(false);
+  };
+
+  return (
+    <div>
+      <IconButton color="primary" onClick={handleMenu}>
+        {props.children}
+      </IconButton>
+
+      <Menu
+        anchorEl={anchorEl}
+        anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
```

Comment:
{ vertical: 'top', horizontal: 'right' } this object you can move to variable, because you use it twice